README.md

```diff
@@ -240,32 +240,7 @@ make setup-dev
 
 Development targets are specified in the [Makefile](./Makefile).
 

```

Comment:
Abdullah mentioned he meant for that ticket to include fix for typos, upper/lower case standardisation, etc. Dunno if you wanna include that.

Reply:
These are covered in the second commit (to the Makefile)

Reply:
I'm going to do a quick pass over the README to check for any other typos